hack/update-codegen.sh

```diff
@@ -109,3 +109,6 @@ ${REPO_ROOT_DIR}/hack/update-openapigen.sh
 
 # Make sure the generated API reference docs are up-to-date
 ${REPO_ROOT_DIR}/hack/update-reference-docs.sh
+
+# Add structural OpenAPI schema to Tekton CRDs
+${REPO_ROOT_DIR}/hack/update-schemas.sh
```

Comment:
After running this script, we need to verify that the generated CRDs have not been modified. 
The `verify-codegen.sh` script achieves this for the other code gens, but it does not include the `config` folder, so it needs to be extended, otherwise we risk missing schema updates.